www/main.js

```diff
@@ -13417,6 +13412,7 @@ module.exports = {
   manage: self.manage,
   notify: self.notify,
   doList: self.list,
+  getUID: self.getUID,
```

Comment:
this is from feedhenry js sdk https://github.com/feedhenry/fh-js-sdk/commit/c0da98314edee6f41c2e920cfd5d9314a00b99c2. Not used on the template